L1Trigger/Phase2L1ParticleFlow/python/l1tMHTPFProducer_cfi.py

```diff
@@ -3,5 +3,5 @@
 l1tMHTPFProducer = cms.EDProducer("L1MhtPfProducer",
                                  jets = cms.InputTag("l1tSCPFL1PuppiEmulator"),
                                  minJetPt = cms.double(30.0),
-                                 maxJetEta = cms.double(2.4)
+                                 maxJetEta = cms.double(550 * cms.math.pi / 720) # 2.4 rounded to hardware unit granularity
```

Comment:
`cms.math.pi` looks suspicious (do we have our own definition of `pi`?)
perhaps you can add `import math` at the top and use `math.pi`, but eventually I think it would be better to have here the float value and make sure it's converted to the right hardware value in the code (e.g. using `l1ct::Scales::makeGlbEta` or  `l1ct::Scales::makeGlbEtaRoundEven`).


For consistency with the firmware you can have also the same float 2.4 in the firmware c++, e.g. 

```cpp
const l1ct::glbeta_t MAX_JET_ETA = l1ct::Scales::makeGlbEta(2.4);
```
as Vitis HLS should be able to evaluate it at compile time and synthesize just the integer.

Reply:
The firmware was actually already like that, so I've changed the emulator to make the cuts in hardware units. Now the eta cut is specified as `2.4` in the configuration, and in the C++ `l1ct::Scales::makeGlbEta` is used to make it match the firmware.